<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>data</class>
 <widget class="QDialog" name="data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>加载数据集</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="cat">
     <item>
      <widget class="QLabel" name="lbl_cat">
       <property name="text">
        <string>猫：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_cat_path">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cat_load">
       <property name="text">
        <string>加载数据集</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dog">
     <item>
      <widget class="QLabel" name="lbl_dog">
       <property name="text">
        <string>狗：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_dog_path">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_dog_load">
       <property name="text">
        <string>加载数据集</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
